web/docs/auth/Pills.css

```diff
@@ -1,6 +1,7 @@
 :root {
   --auth-pills-color: #333;
   --auth-pills-email: #e0f2fe;
+  --auth-pills-discord: #f1f5f9;
```

Comment:
Wasn't sure what color to use for these, copilot picked this one :^)

Reply:
We like to be more intentional about the color choices :D Let's go for `#d3d6f2` for light and `#2f3670` for dark variant.